kickoff/templates/includes/firefox_release.html

```diff
@@ -1,7 +1,8 @@
 <script type='text/javascript'>
+URLToPartial="//relman-vm1.dmz.scl3.mozilla.com/partial.json";
```

Comment:
@JeremiePat confirmed that this feature is quite old (~ie5) but not well known. Anyway, for once, glad I told you something you didn't know :)